libkbfs/backpressure_disk_limiter.go

```diff
@@ -106,13 +106,24 @@ func (bt backpressureTracker) currLimit() float64 {
 	usedFloat := float64(bt.used)
 	freeFloat := float64(bt.free)
 	limit := bt.limitFrac * (usedFloat + freeFloat)
-	return math.Min(limit, float64(bt.limit))
+	minLimit := math.Min(limit, float64(bt.limit))
+	// The magic number of 512 gets us past overflow issues at the limit due to
```

Comment:
Ok, I thought you might have found an algorithmic explanation somewhere.  Maybe just put a "(according to local tests)" somewhere?